### PR TITLE
[OSCD] Added a rule to detect abusing windows telemetry for persistence

### PR DESCRIPTION
**Pull request by zipa-original**

### CODE REVIEW

**Comment by aw350m33d** [comment]
Hi Aleksey! Thanks for participating in the sprint!

Could you correct the `title`? According to the recommendations of the maintainers, in the `title`, you need to write all words with a capital letter, except for prepositions and conjunctions.

File: rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title: Registry Persistence Mechanism via Windows Telemetry 
+id: 73a883d0-0348-4be4-a8d8-51031c2564f8
+description: Detects persistence method using windows telemetry 
+status: experimental
+date: 2020/10/16
+references:
+    - https://www.trustedsec.com/blog/abusing-windows-telemetry-for-persistence/
+author: Lednyov Alexey, oscd.community
+tags:
+    - attack.persistence
```
Comment (yugoslavskiy) [inline rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml:10]: Helllo @zipa-original! Thank you for your contribution! Could you please add a technique tag as well?

Comment (zipa-original) [reply rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml:10]: Hello! Unfortunately, I could not find any suitable technique in the ATT&CK matrix. That's why I decided to leave only tactics.

Comment (yugoslavskiy) [reply rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml:10]: I believe that you can use `T1053.005` (Scheduled Task)  as it's gonna be executed by it.

**Comment by zipa-original** [comment]
Hi, @aw350m33d! I fixed the title.

File: rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml (hunk 1 above)
Comment (zipa-original) [reply rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml:10]: I thought about it. But, there is no creation or modification tasks... Do you think it's better to add anyway?

Comment (yugoslavskiy) [reply rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml:10]: "Kinda close" context is better than the absence of context (:

Comment (yugoslavskiy) [reply rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml:10]: You can contribute to the ATT&CK and create the proper technique and then update the rule. What do you think about it?

Comment (yugoslavskiy) [reply rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml:10]: *but for now you can just add `T1053.005` here, @zipa-original

Comment (zipa-original) [reply rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml:10]: Ok

**Comment by yugoslavskiy** [comment]
Thank you very much, @zipa-original! Great contribution!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**